File: 프로그래머스/0/120819. 아이스 아메리카노/아이스 아메리카노.js
function solution(money) {
    // 아이스 아메리카노 한 잔 가격
    const price = 5500;
    
    // 최대로 마실 수 있는 잔 수
    const maxCups = Math.floor(money / price);
    
    // 남는 돈
    const remainingMoney = money % price;
    
    // 결과 배열
    return [maxCups, remainingMoney];
}

// 예시 호출
console.log(solution(11000)); // [2, 0]
console.log(solution(15000)); // [2, 4000]

File: 프로그래머스/0/120819. 아이스 아메리카노/아이스 아메리카노.py
def solution(money):
  answer = []
  item = money // 5500 # 5500원으로 몇 개의 아이템을 살 수 있을 지 계산
  item2 = money % 5500 # 나머지 돈 계산
  answer = [item, item2]
  return answer